<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addnewpartywindow</class>
 <widget class="QDialog" name="addnewpartywindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 74, 106);</string>
  </property>
  <widget class="QTextEdit" name="addNewPartyName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 97, 138);
border-radius:15px;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="addNewPartyDescribe">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>481</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 97, 138);
border-radius:15px;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="addNewPartyConnection">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>481</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 97, 138);
border-radius:15px;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>530</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 181, 205);
color: rgb(255, 255, 255);
border: 0px;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>530</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 181, 205);
color: rgb(255, 255, 255);
border: 0px;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="peoNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="peoDescLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>介绍：</string>
   </property>
  </widget>
  <widget class="QLabel" name="peoConnecLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>关系：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
